src/funding/card/config.jsx

```diff
@@ -122,10 +122,20 @@ export function getCardConfig() : FundingSourceConfig {
         },
 
         Label: ({ logo, locale, content, custom }) => {
-            if (custom && custom.label) {
+            if (custom) {
+                const validLabels = {
+                    checkout: 'Checkout'
+                };
+
+                let label = 'Checkout';
```

Comment:
Let's reuse validLabels for the default value:
```suggestion
                let label = validLabels.checkout;
```